server/controllers/conversation.js

```diff
@@ -0,0 +1,113 @@
+const Conversation = require('../models/Conversation');
+const asyncHandler = require('express-async-handler');
+
+// @route POST /create-conversation/
+// @desc create a new conversation
+// @access Public
+
+exports.createConversation = asyncHandler(async (req, res, next) => {
+  const { description, receiver } = req.body;
+
+  if (!description || !receiver) {
+    res.status(400);
+    throw new Error('Bad request! Missing description or receiver!');
+  }
+
+  const conversation = await Conversation.create({
```

Comment:
Let's also perform a check to see if a conversation between the two users already exists before we create a new one

Reply:
Okay I added a check and now conversation will just be pushed with a new message if it exists